src/components/HomeLayout/HomeLayout.js

```diff
@@ -18,42 +17,40 @@ const useStyles = makeStyles({
   toolbar: {
     position: "static",
     top: 0
-  },
-  menuIcon: {
-    fontSize: "2.24em",
-    color: "rgba(0, 0, 0, .54)",
-    display: "none" //temporarily disabled
   }
 });
 
 const HomeLayout = props => {
   const { children, bg, activeStep = 1, forceLogoDisplay } = props;
   const classes = useStyles(props);
+
   const bgUrl = bg[activeStep] ? bg[activeStep] : bg[bg.length - 1];
 
   const handleLogoOnClick = e => {
     window.location.href = "/";
   };
 
   return (
-    <Grid
-      container
-      alignContent="center"
-      alignItems="center"
-      className={`${classes.container} HomeLayout`}
-      style={{ backgroundImage: `url(${bgUrl})` }}
-    >
-      <Grid container style={{ height: "100%" }}>
-        <Grid container direction="column">
-          <Toolbar className={classes.toolbar}>
-            {(activeStep !== 1 || forceLogoDisplay) && (
-              <LogoMini style={{ flex: 1 }} onClick={handleLogoOnClick} />
-            )}
-          </Toolbar>
+    <>
```

Comment:
What does this mean?

Reply:
It's a short syntax for a fragment: https://reactjs.org/docs/fragments.html#short-syntax
typically used when you want to wrap stuff in a render

Reply:
Didn't know, nice stuff